src/applications/vaos/project-cheetah/components/SelectDate2Page.jsx

```diff
@@ -1,46 +1,238 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { connect } from 'react-redux';
 import * as actions from '../redux/actions';
 import FormButtons from '../../components/FormButtons';
 import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import CalendarWidget from '../../components/calendar/CalendarWidget';
+import moment from 'moment';
+import { FETCH_STATUS } from '../../utils/constants';
+import { getDateTimeSelect } from '../redux/selectors';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { getRealFacilityId } from '../../utils/appointment';
 
 const pageKey = 'selectDate2';
-const pageTitle = 'Select Second Date';
+const pageTitle = 'Select First Date';
 
-function SelectDate2Page({
-  pageChangeInProgress,
+const missingDateError =
+  'Please choose your preferred date and time for your appointment.';
+
+export function getOptionsByDate(
+  selectedDate,
+  timezoneDescription,
+  availableSlots = [],
+) {
+  return availableSlots.reduce((acc, slot) => {
+    if (slot.start.split('T')[0] === selectedDate) {
+      let time = moment(slot.start);
+      if (slot.start.endsWith('Z')) {
+        time = time.tz(timezoneDescription);
+      }
+      const meridiem = time.format('A');
+      const screenReaderMeridiem = meridiem.replace(/\./g, '').toUpperCase();
+      acc.push({
+        value: slot.start,
+        label: (
+          <>
+            {time.format('h:mm')} <span aria-hidden="true">{meridiem}</span>{' '}
+            <span className="sr-only">{screenReaderMeridiem}</span>
+          </>
+        ),
+      });
+    }
+    return acc;
+  }, []);
+}
+
+function ErrorMessage({ facilityId, requestAppointmentDateChoice }) {
```

Comment:
You can remove `requestAppointmentDateChoice` and any uses of it in this message and in the actions. Users won't have the option of dropping into the request flow, most likely.